resallocserver/manager.py

```diff
@@ -298,10 +306,63 @@ def job(self):
         self.event.set()
 
 
+class CleanUnknownWorker(Worker):
+    """
+    Delete all resources that are not recognized by resalloc
+    https://github.com/praiskup/resalloc/issues/88
+    """
+    def job(self):
+        if not self.pool.cmd_list:
+            return
+
+        all_resources = self._list_all_resources()
+        known_resources = self._list_known_resources()
+        unknown_resources = set(all_resources) - set(known_resources)
+
+        # There is no assigned resource ID for the given resource, that's
+        # the whole point. Let's simply use 0, which means the termination
+        # output for all unknown resources will be logged into
+        # hooks/000000_terminate
```

Comment:
Well in theory we could deduct the resource ID from its name, but I don't object here.